docs/static/config-details.asciidoc

```diff
@@ -118,6 +118,29 @@ To summarize, we have 3 categories of memory usage, where 2 can be limited by th
 
 Keep these memory requirements in mind as you calculate your ideal memory allocation.
 
+[[reducing-off-heap-usage]]
```

Comment:
I wonder if we should better highlight this somehow. The use of this setting completely changes the way Logstash works. Also, it would give us the ability to better understand Logstash when troubleshooting.

However, the direct result of this is that will completely impact/affect JVM Heap sizing. So I propose we:

1. Bring some more visibility into this setting, maybe say this will be the default in future versions of Logstash.
2. Tell users that this is a recommended setting to be set?
3. And, in the memory size/performance tuning sections to have references to this setting being turned on??

Any thoughts?

@jsvd @flexitrev